Sources/AudioKit/MIDI/MIDINode.swift

```diff
@@ -14,7 +14,7 @@ open class MIDINode: Node, MIDIListener, NamedNode {
     open var midiIn = MIDIEndpointRef()
 
     /// Name of the instrument
-    open var name = "MIDINode"
+    open var name: String? = "MIDINode"
```

Comment:
Would not make this optional. See previous comments for rationale.

Reply:
AudioKit 5 did away with as many optionals as we could. Optionals have their purpose but they should be optional for a reason, not just by default.